src/v2i-hub/MobilityOperationPlugin/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name":"MobilityOperationPlugin",
+	"description":"...",
+	"version":"5.0.0",
+	"exeLocation":"/bin/MobilityOperationPlugin",
+	"coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+	"messageTypes":[
+		{
+			"type":"J2735",
+			"subtype":"TMSG03-P",
+			"description":"..."
+		}
+	],
+	"configuration":[
```

Comment:
Might need some more config parameters.

Reply:
There are follow on issues for this plugin that I haven't looked at but in terms of the functionality required for this issue I am not sure what config parameters would currently be required